doc/man3/EVP_PKEY_fromdata.pod

```diff
@@ -120,24 +120,19 @@ TODO Write a set of cookbook documents and link to them.
 =head2 Creating an ECC keypair using raw key data
 
  #include <openssl/evp.h>
+ #include <openssl/param_build.h>
 
  /*
-  * These arrays represent large numbers, big endian organization.
-  * In a real application, these would probably be bignums that get
-  * converted to the native integer organization with BN_bn2nativepad().
-  * We're not doing that here, since this is not an example of BIGNUM
-  * functionality, but an example of EVP_PKEY_fromdata().
+  * Fix data to represent the private and public key.  The private
+  * is passed as an integer, while the public key is an octet string.
```

Comment:
I think describing it as a integer here for the user is a bit confusing..
It looks like a BN..

Reply:
For all intents and purposes, a BIGNUM is an integer.  I would explain it with ℤ if there weren't for compilers not quite liking UTF-8 in comments...

Reply:
I think that the easiest is to remove all but the first sentence of that comment

Reply:
I know what you are saying and how it is implemented, but just think of your poor end user.. The thing we actually input in a BIGNUM represented as a sequence of bytes.
THAT WILL BE CONFUSING TO A NEW USER.

Reply:
I typed the above before seeing your reply.